truckersmp-cli

```diff
@@ -1,512 +1,547 @@
     # We can detect the timestamp update with inotifywait -e modify.
     # Needs inotify-tools installed
-    if ! pidof steam; then
-        print "Starting Steam…"
-        nohup steam > /dev/null 2>&1 &
-
-        # Wait for user login
-        if command -v inotifywait > /dev/null 2>&1; then
-            print "Waiting for Steam using inotifywait."
-            inotifywait -e modify -qq "$XDG_DATA_HOME/Steam/config/loginusers.vdf"
-            print "Steam should now be up and running and the user logged in."
-        else
+    try:
+        subprocess.check_call(["pidof", "steam"], stdout = subprocess.DEVNULL)
+    except:
+        printdbg("Starting Steam…")
+        subprocess.Popen(["nohup", "steam"],
+          stdout = subprocess.DEVNULL, stderr = subprocess.STDOUT)
+
+        # wait for user login
+        try:
+            if not distutils.spawn.find_executable("inotifywait"):
+                printerr("Inotify-tools not installed,", end = " ")
+                raise Exception
+
+            printdbg("Waiting for Steam using inotifywait.")
+            loginusers_vdf = os.path.join(steamdir, "config/loginusers.vdf")
+            if os.path.isfile(loginusers_vdf):
+                subprocess.call(
+                  ["inotifywait", "-e", "modify", "-qq", loginusers_vdf])
+                printdbg("Steam should now be up and running and the user logged in.")
+            else:
+                printerr("{} is not present,".format(loginusers_vdf), end = " ")
+                raise Exception
+        except:
             # fallback to 30 seconds if inotify-tools is not installed
+            # or config/loginusers.vdf is not present
             # Maybe enhance this with manual timestamp comparison in the future.
-            >&2 echo "Inotify-tools not installed, waiting 30 seconds for steam to start up."
-            sleep 30
-            print "Steam should be up now"
-        fi
-    fi
-
-    if ! [ -d "$prefixdir" ]; then
-        print "mkdir -p $prefixdir"
-        mkdir -p "$prefixdir"
-    fi
-
-    print "Startup command:"
-    print   "SteamGameId=$steamid" \
-            "SteamAppId=$steamid" \
-            "STEAM_COMPAT_DATA_PATH=$prefixdir" \
-            "STEAM_COMPAT_CLIENT_INSTALL_PATH=$XDG_DATA_HOME/Steam" \
-            "/usr/bin/env python3 $protondir/proton" \
-            "run" \
-            "$scriptdir/truckersmp-cli.exe $gamedir $moddir"
-
-    SteamGameId="$steamid" \
-    SteamAppId="$steamid" \
-    STEAM_COMPAT_DATA_PATH="$prefixdir" \
-    STEAM_COMPAT_CLIENT_INSTALL_PATH="$XDG_DATA_HOME/Steam" \
-    /usr/bin/env python3    "$protondir/proton" \
-                            run \
-                            "$scriptdir/truckersmp-cli.exe" \
-                            "$gamedir" \
-                            "$moddir"
-}
-
-start_with_wine() {
-cat <<WINE_USAGE
-
+            printerr("waiting 30 seconds for steam to start up.")
+            time.sleep(30)
```

Comment:
If we want to be fancy, let's add a countdown here. That's just an idea though, not required for me to accept this PR ^^